server/src/bootstrap/index.ts

```diff
@@ -194,9 +198,100 @@ if (baseConfig.nodeEnv === Environment.Prod) {
     path.resolve(__dirname, '../../..', 'client', 'build', 'index.html'),
   )
 
-  app.get('*', (_req, res) =>
-    res.header('content-type', 'text/html').send(index),
-  )
+  app.get('/agency/:shortname', (req, res) => {
+    const $ = cheerio.load(index)
+
+    $('title').text(`${req.params.shortname.toUpperCase()} FAQ - AskGov`)
+    $('meta[property="og:title"]').attr(
+      'content',
+      `${req.params.shortname.toUpperCase()} FAQ - AskGov`,
+    )
+    $('meta[property="og:url"]').attr(
+      'content',
+      `${req.protocol}://${req.hostname}${req.originalUrl.toLowerCase()}`,
+    )
+
+    const agencyPromise = async () => {
+      return await agencyService.findOneByName({
+        shortname: req.params.shortname,
+      })
+    }
+    ;(async () => {
+      const agency = await agencyPromise()
+      if (agency.isOk()) {
+        $('meta[name="description"]').attr(
+          'content',
+          `Answers from ${
+            agency.value.longname
+          } (${req.params.shortname.toUpperCase()})`,
+        )
+        $('meta[property="og:description"]').attr(
+          'content',
+          `Answers from ${
+            agency.value.longname
+          } (${req.params.shortname.toUpperCase()})`,
+        )
+      } else {
+        $('meta[name="description"]').attr(
+          'content',
+          `Answers from ${req.params.shortname.toUpperCase()}`,
+        )
+        $('meta[property="og:description"]').attr(
+          'content',
+          `Answers from ${req.params.shortname.toUpperCase()}`,
+        )
+      }
+      res.header('content-type', 'text/html').send($.html())
+    })()
```

Comment:
Why wrap it up like this? You can simply make the `(req, res) => { ... }` handler async